src/js/components/reviewData/ReviewDataPage.jsx

```diff
@@ -30,6 +31,11 @@ export default class ReviewDataPage extends React.Component {
             warningMessage = <PublishedSubmissionWarningBanner />;
         }
 
+        let gtasWarning = null;
+        if(this.props.data.gtas && this.props.data.gtas.open) {
```

Comment:
`gtas.open` no longer exists. Given the suggestion below you can probably just do the first part of this `if` for checking.